snippets/post_install_network_config

```diff
@@ -77,6 +77,9 @@ mv /etc/sysconfig/network.cobbler /etc/sysconfig/network
 # Also set the hostname now, some applications require it
 # (e.g.: if we're connecting to Puppet before a reboot).
 /bin/hostname $hostname
+        #if $osversion == "rhel7"
```

Comment:
Using 'hostnamectl set-hostname $hostname' was my first try. I followed the instructions on https://access.redhat.com/solutions/1452273 which say:

```
%post --nochroot
hostnamectl set-hostname server.example.com
hostnamectl --pretty set-hostname server.example.com
cp /etc/hostname /mnt/sysimage/etc/hostname
cp /etc/machine-info /mnt/sysimage/etc/machine-info
```

But this was not working, because **post_install_network_config** is executed in **_%post_** instead of **_%post --nochroot_**.